harmony/harmony.py

```diff
@@ -375,6 +394,52 @@ def error_messages(self) -> List[str]:
         return spatial_msgs + temporal_msgs + shape_msgs + dimension_msgs + parameter_msgs
 
 
+class CapabilitiesRequest(BaseRequest):
+    """A Harmony request to get the harmony capabilities of a CMR collection
+    Args:
+        A dict with optional collectionId, shortName and capabilities_version fields
```

Comment:
I think it would be more idiomatic Python to just use keyword arguments here instead of taking a dictionary as a parameter